content/documentation/pages/1-installation/3-kubernetes/1-helm.md

```diff
@@ -1,422 +1,10 @@
-> `SPRING_CLOUD_SKIPPER_CLIENT_SERVER_URI` property in
-> `src/kubernetes/server/server-deployment.yaml` should point to it.
-
-The Data Flow Server uses the [Fabric8 Java client
-library](https://github.com/fabric8io/kubernetes-client) to connect to
-the Kubernetes cluster. We use environment variables to set the values
-needed when deploying the Data Flow server to Kubernetes. We also use
-the [Spring Cloud Kubernetes
-library](https://github.com/spring-cloud/spring-cloud-kubernetes) to
-access the Kubernetes
-[`ConfigMap`](https://kubernetes.io/docs/user-guide/configmap/) and
-[`Secrets`](https://kubernetes.io/docs/user-guide/secrets/) settings.
-The `ConfigMap` settings for RabbitMQ are specified in the
-`src/kubernetes/server/server-config-rabbit.yaml` file and for Kafka in
-the `src/kubernetes/server/server-config-kafka.yaml` file. MySQL secrets
-are located in the `src/kubernetes/mysql/mysql-secrets.yaml` file. If
-you modified the password for MySQL, you should change it in the
-`src/kubernetes/mysql/mysql-secrets.yaml` file. Any secrets have to be
-provided in base64 encoding.
-
-> **Note**
->
-> We now configure the Data Flow server with file-based security, and
-> the default user is _user_ with a password of _password_. You should
-> change these values in
-> `src/kubernetes/server/server-config-rabbit.yaml` for RabbitMQ or
-> `src/kubernetes/server/server-config-kafka.yaml` for Kafka.
-
-To create the configuration map when using RabbitMQ, run the following
-command:
-
-    kubectl create -f src/kubernetes/server/server-config-rabbit.yaml
-
-To create the configuration map when using Kafka, run the following
-command:
-
-    kubectl create -f src/kubernetes/server/server-config.yaml
-
-Now you need to create the server deployment, by running the following
-commands:
-
-    kubectl create -f src/kubernetes/server/server-svc.yaml
-    kubectl create -f src/kubernetes/server/server-deployment.yaml
-
-You can use `kubectl get all -l app=scdf-server` to verify that the
-deployment, pod, and service resources are running. You can use
-`kubectl delete all,cm -l app=scdf-server` to clean up afterwards.
-
-You can use the `kubectl get svc scdf-server` command to locate the
-`EXTERNAL_IP` address assigned to `scdf-server`. You can use that
-address later to connect from the shell. The following example (with
-output) shows how to do so:
-
-    $ kubectl get svc scdf-server
-    NAME         CLUSTER-IP       EXTERNAL-IP       PORT(S)    AGE
-    scdf-server  10.103.246.82    130.211.203.246   80/TCP     4m
-
-In this case, the URL you need to use is `https://130.211.203.246`.
-
-If you use Minikube, you do not have an external load balancer, and the
-`EXTERNAL_IP` shows as `<pending>`. You need to use the `NodePort`
-assigned for the `scdf-server` service. You can use the following
-command to look up the URL to use:
-
-    $ minikube service --url scdf-server
-    https://192.168.99.100:31991
+## Install using Helm
```

Comment:
Should we focus on only one installation option either helm or kubectl?   I'm thinking for simplicity.   A.   i.e. we use kubectl since its the easiest against minikube.   And refer them to the helm chart instructions if they need it.

Reply:
If you are ok with installing helm, that is the easiest way, so I would prefer promoting that approach.  I don't want to get rid of the kubectl option either - in fact we may make it much easier by adding in skaffold and using the features in kustomize/kubectl.